README.md

```diff
@@ -35,9 +35,12 @@ To get started, clone down the repository using:
 
     git clone https://github.com/Senither/hypixel-discord-chat-bridge.git
 
-Next go into the `hypixel-discord-chat-bridge` folder and install all the dependencies using Yarn.
+Next go into the `hypixel-discord-chat-bridge` folder and install all the dependencies using npm.
 
-    yarn
+    npm install yarn.lock
+    npm uninstall mineflayer
+    npm install mineflayer
+    npm audit fix
```

Comment:
Not sure what the need to switch from yarn to npm for dependencies is, and even if switching this isn't the way to do it.